src/components/nav-bar/index.tsx

```diff
@@ -1,157 +1,147 @@
-import BallotIcon from '@mui/icons-material/Ballot';
-import InventoryIcon from '@mui/icons-material/Inventory';
-import EqualizerIcon from '@mui/icons-material/Equalizer';
-import GroupIcon from '@mui/icons-material/Group';
+import { useState } from 'react';
+import { Box, Button, Divider, Collapse } from '@mui/material';
+import {
+  AddBox,
+  Assignment,
+  Email,
+  Equalizer,
+  ExpandLess,
+  ExpandMore,
+  Group,
+  HandshakeRounded,
+  Inventory,
+  MiscellaneousServices,
+  VolunteerActivism,
+} from '@mui/icons-material';
+import Image from 'next/image';
 
 export default function Navbar() {
+  const [expandedSection, setExpandedSection] = useState<string | null>(null);
+
+  // Expand or Collapse sub-buttons
+  const handleToggle = (section: string) => {
+    setExpandedSection((prev) => (prev === section ? null : section));
+  };
+
+  const buttonStyles: React.CSSProperties = {
+    color: '#ff8a65',
+    fontSize: '15px',
+    textTransform: 'none',
+    justifyContent: 'flex-start',
+    width: '100%',
+    padding: '10px 10px',
+  };
+
+  const sections = [
+    {
+      title: 'Donations',
+      icon: <VolunteerActivism />,
+      buttons: [
+        {
+          href: '/donation/add',
+          icon: <AddBox />,
+          label: 'Add Donation',
+        },
+        {
+          href: '/donation',
+          icon: <VolunteerActivism />,
+          label: 'Donations',
+        },
+        { href: '/donors', icon: <HandshakeRounded />, label: 'Donors' },
+      ],
+    },
+    {
+      title: 'Inventory',
+      icon: <Inventory />,
+      buttons: [
+        { href: '/donationItem', icon: <Inventory />, label: 'Inventory' },
+        { href: '/item/add', icon: <AddBox />, label: 'Add Evaluation' },
+        { href: '/item', icon: <Assignment />, label: 'Evaluations' },
+      ],
+    },
+    {
+      title: 'Reporting & Settings',
+      icon: <MiscellaneousServices />,
+      buttons: [
+        { href: '/mailMerge', icon: <Email />, label: 'Emails' },
+        { href: '/user', icon: <Group />, label: 'Users' },
+        { href: '#', icon: <Equalizer />, label: 'Reports' },
+      ],
+    },
+  ];
+
   return (
-    <>
-      <Box height={'100vh'}>
-        <Box display={'flex'} justifyContent={'center'} p={2}>
+    <Box
+      height="100vh"
+      display="flex"
+      flexDirection="column"
+      width="33vh" // Fix width of the navbar
+      overflow="hidden" // Prevent any expanding overflow
+      borderRight="1px solid #ddd"
+    >
+      {/* Logo Section */}
+      <Box display="flex" justifyContent="center">
+        <Button href="/">
           <Image
-            className="rounded-md"
             alt="logo"
             src="/cropped-MOH-Logo-768x393.png"
-            width="180"
-            height="100"
-            // style={{ marginRight: '25px' }} // Add spacing between the image and links
+            width="220"
+            height="107"
```

Comment:
Hardcoded sizes are best to be avoided. If I zoom in on my screen, the image gets cut off. Possible to have the size be dynamic? 

![image](https://github.com/user-attachments/assets/77717049-4565-4129-baf8-aa2a53481301)

Reply:
I fixed this one. The image should be responsive now!

Reply:
I also fixed the text to make it responsive too!